logprep/processor/generic_adder/processor.py

```diff
@@ -27,21 +21,55 @@ class DuplicationError(GenericAdderError):
 
     def __init__(self, name: str, skipped_fields: List[str]):
         message = (
-            "The following fields already existed and " "were not overwritten by the GenericAdder: "
+            "The following fields already existed and were not overwritten by the GenericAdder: "
         )
         message += " ".join(skipped_fields)
 
         super().__init__(name, message)
 
 
 class GenericAdder(RuleBasedProcessor):
```

Comment:
We have already discussed this so just for documentation purposes: This has to be changed to the new interface:

```suggestion
class GenericAdder(Processor):
```

With that all the other changes according to the new interface has to be done as well.

Reply:
I have just merged main into this soon after you wrote this. So this should be done now.